docker-compose-dev.yml

```diff
@@ -25,8 +25,6 @@ services:
     ports:
      - "${WDQS_FRONTEND_PORT}:80"
     restart: ${RESTART}
```

Comment:
This will only work is wdqs_frontend depends on wikibase. I added that somewhere but now it's gone. Perhaps it would be better to make it into a separate issue/patch